### PR TITLE
Fix #3698: Introduce new issue forms for Android. Add newsletter and donate link to README. Add explanatory note to the contributor wiki page.

### LINKED ISSUE #3698: Use GitHub issues forms
See https://dev.to/github/how-to-build-google-like-forms-with-github-3ig2. GitHub now has support for filing structured issues rather than having to pre-fill parts of Markdown. This seems like a nice, straightforward change that we could do to make filing issues nicer.

### PR DESCRIPTION
**Pull request by seanlip**
## Explanation

This PR fixes #3698 by introducing new issue forms for Android, following the guidance here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
It also adds links to our newsletter and donate pages to the README, and adds an explanatory note to the contributor wiki page (since the animated image is sometimes slow to load).

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2_feature_request_form.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a bug in the Oppia project.
+  - type: markdown
+    attributes:
+      value: |
+        Before filing a new issue, **please do a quick search** to check that it hasn't already been filed on the issue tracker. You can do this by going to [this link](https://github.com/oppia/oppia-android/issues) and typing some words related to the issue in the search box next to the "New issue" button.
+  - type: textarea
+    id: problem
```
Comment (adhiamboperes) [inline .github/ISSUE_TEMPLATE/2_feature_request_form.yml:15]: Why is it that the fields of the feature request forms   have an id, but the  bug reports don't?

Comment (seanlip) [reply .github/ISSUE_TEMPLATE/2_feature_request_form.yml:15]: That's a good question -- I'm not sure, I copied the feature template from the Oppia Web version. The [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) suggest having an id (though I'm not sure what it's used for), so I've gone ahead and added IDs to the bug report form as well. Thanks for catching!

**Comment by seanlip** [comment]
@MohitGupta121 @adhiamboperes PTAL. Thanks!

**Review by MohitGupta121 — APPROVED** [review]
@seanlip LGTM, that's really awesome feature of newsletter. Now we have active community. Thanks!

File: .github/ISSUE_TEMPLATE/2_feature_request_form.yml (hunk 1 above)
Comment (adhiamboperes) [reply .github/ISSUE_TEMPLATE/2_feature_request_form.yml:15]: I did some digging around and found that these may be used in a scenario like the one Ben described in #5043

**Review by adhiamboperes — APPROVED** [review]
This LGTM. Thanks @seanlip!

**Comment by seanlip** [comment]
I'm going to go ahead and merge this since Ben's not around. Thanks for the reviews @adhiamboperes and @MohitGupta121!

**Pull request merged**